src/cephadm/cephadm

```diff
@@ -119,6 +120,19 @@ class EndPoint:
         return f'{self.ip}:{self.port}'
 
 
+class ContainerInfo:
```

Comment:
I'm not sure if the python code in ceph has a pattern of using or not using dataclasses. I def. won't insist on it but this certainly looks like something that calls out to be a dataclass.

https://docs.python.org/3/library/dataclasses.html

Reply:
AFAIK we are not using `dataclasses` in the mgr module (yet). Probably because they require `Python 3.7` and we are assuming a lower version of Python, but I'm not sure.